Checkbox/Checkbox.jsx

```diff
@@ -13,6 +13,9 @@ export default class Checkbox extends MaterialComponent {
   componentDidMount() {
     this.MDComponent = new MDCCheckbox(this.control);
   }
+  componentWillUnmount() {
+    this.MDComponent.destroy();
```

Comment:
since destroy is something that is not publicly documented, do you think it is better to safe guard it with an `&&` condition?

```js
this.MDComponent.destroy && this.MDComponent.destroy();
```